src/SDKs/_metadata/datafactory_resource-manager.txt

```diff
@@ -1,11 +1,10 @@
-2018-02-02 07:13:12 UTC
+2018-02-03 03:06:57 UTC
 
 1) azure-rest-api-specs repository information
 GitHub user: Azure
 Branch:      master
-Commit:      f9ec24015ab77a2d7a1dfd33ddd208824459a7ca
 
 2) AutoRest information
 Requested version: latest
-Bootstrapper version:    C:\Program Files\nodejs `-- autorest@2.0.4245  
+Bootstrapper version:    C:\Users\yanzhang\AppData\Roaming\npm `-- autorest@2.0.4245  
```

Comment:
@zhangyd2015 please make sure the commit is part of the meta data file.
This means you are not generating it from the right repo.

Reply:
It turned out the bugs in Windows 10 RS4. I changed another machine and the commit id was generated correctly.